src/app/components/ATIAnalytics/atiUrl/index.ts

```diff
@@ -490,10 +492,11 @@ export const buildReverbAnalyticsModel = ({
           x16: aggregatedCampaigns,
           x17: categoryName,
           x18: isLocServeCookieSet(),
-          ...(experimentVariant && {
-            mv_test: OPTIMIZELY_CONFIG.flagKey,
-            mv_creation: `${experimentVariant}`,
-          }),
+          ...(experimentVariant &&
+            experimentName && {
+              mv_test: experimentName,
+              mv_creation: `${experimentVariant}`,
```

Comment:
Do we need to stringify the value?
```suggestion
              mv_creation: experimentVariant,
```